runtimes/pythonrt/py-sdk/autokitteh/discord.py

```diff
@@ -33,3 +33,13 @@ def discord_client(connection: str, intents=None, **kwargs) -> discord.Client:
         raise ConnectionInitError(connection)
 
     return discord.Client(intents=intents, **kwargs)
+
+
+def get_bot_token(connection: str):
```

Comment:
Nit: please rename to just `bot_token`. Calling `discord.bot_token("conn_name")` feels slightly better, given that reading the value is so trivial.